frontend/components/TableContainer/DataTable/HostMdmStatusCell/HostMdmStatusCell.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import ReactTooltip from "react-tooltip";
+import { COLORS } from "styles/var/colors";
+import { DEFAULT_EMPTY_CELL_VALUE } from "utilities/constants";
+import Icon from "components/Icon";
+import NotSupported from "components/NotSupported";
+import { IHost } from "interfaces/host";
+
+const baseClass = "host-mdm-status-cell";
+
+const HostMdmStatusCell = ({
+  row: {
+    original: { id, mdm, platform },
+  },
+  cell: { value },
+}: {
+  row: { original: IHost };
+  cell: { value: string };
+}): JSX.Element => {
+  if (platform === "chrome") {
+    return NotSupported;
+  }
+
+  if (!value) {
+    return <span className={`${baseClass}`}>{DEFAULT_EMPTY_CELL_VALUE}</span>;
+  }
+
+  return (
+    <span className={`${baseClass}`}>
+      {value}
+      {mdm?.dep_profile_error && (
+        <>
+          <span
+            className={`tooltip tooltip__tooltip-icon`}
+            data-tip
+            data-for={`host-mdm-status__${id.toString()}`}
```

Comment:
nit: since you're doing interpolation here, calling `toString` is redundant.

Reply:
Nice catch, looks like this was carried over from the other components in the parent directory. I'll make a note to clean those up as we go.